avatar.css

```diff
@@ -0,0 +1,118 @@
+@import url('https://fonts.googleapis.com/css2?family=Raleway:wght@400;500;600&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Raleway', sans-serif;
+}
+
+:root {
+    --primary-color: #064e3b;
+    --text-color: #fff;
+    --border-color: #fff;
+    --green-badge: #059669;
+    --red-badge: #dc2626;
+    --yellow-badge: #facc15;
+}
+
+.container {
+    margin: 2rem;
+    display: flex;
+    flex-wrap: wrap;
+    align-items: flex-end;
+    gap: 1rem;
+}
+
+
+.avatar {
+    border-radius: 50%;
+}
+
+
+.avatar-large-size {
+    width: 8rem;
+    height: 8rem;
+    font-size: 3rem;
+}
+
+.avatar-medium-size {
+    width: 5rem;
+    height: 5rem;
+    font-size: 1.5rem;
+}
+
+.avatar-small-size {
+    width: 3rem;
+    height: 3rem;
+    font-size: .875rem;
+}
```

Comment:
You can use standard conventions for sizing e.g avatar-sm, avatar-lg.

Reply:
Ok, I  thought to name it in this way so that it's more readable. I didn't knew about the standards, Thank you.